app/build.gradle

```diff
@@ -13,7 +13,7 @@ android {
 
     defaultConfig {
         applicationId "com.todoist_android"
-        minSdk 21
+        minSdk 26
```

Comment:
why bump it to 26 ?

Reply:
need to downgrade that, but LocalDateTime.now() requires API level 26+ as a minSdk